addons/udes_stock/controllers/stock_location.py

```diff
@@ -71,3 +71,46 @@ def pi_count(self, pi_request):
         location = Location.get_location(location_id)
 
         return location.process_perpetual_inventory_request(pi_request)
+
+    @http.route('/api/stock-location/<ident>/block/',
```

Comment:
Could use '/api/stock-location/<int:ident>/block': this would automatically perform the validation of ident as a non-empty integer.

The trailing slash on the URI seems incorrect: is there a reason for it?